docs/user/ml/index.asciidoc

```diff
@@ -165,4 +165,35 @@ Select a field for categorization and optionally apply any filters that you
 want, then start the analysis. The analysis uses the same algorithms as a {ml} 
 categorization job. The results of the analysis are shown in a table that makes 
 it possible to open **Discover** and show or filter out the given category 
-there, which helps you to further examine your log messages.
\ No newline at end of file
+there, which helps you to further examine your log messages.
+
+
+[discrete]
+[[change-point-detection]]
+=== Change point detection
+
+preview::[]
+
+Change point detection uses the 
+{ref}/search-aggregations-change-point-aggregation.html[change point aggregation] 
+to detect distribution changes, trend changes, and other statistically 
+significant change points in a metric of your time series data.
+
+You can find change point detection under **{ml-app}** > **AIOps Labs** where 
+you can select the {data-source} or saved search that you want to analyze.
+
+[role="screenshot"]
+image::user/ml/images/ml-change-point-detection.png[Change point detection UI]
+
+Select a function and a metric field, then pick a date range to start detecting 
+change points in the defined range. Optionally, you can split the data by a 
+field with a maximum cardinality of 10,000.
```

Comment:
I wonder if the last sentence might seem confusing, because it's possible to split by fields with a higher cardinality, we just limit the analysis to the first 10,000 values.

Reply:
Thanks, I fixed it in [68f086e](https://github.com/elastic/kibana/pull/151337/commits/68f086ed8797fc61e955c9ec148b88f9575d3db7).